sormas-api/src/main/resources/strings.properties

```diff
@@ -220,6 +220,7 @@ confirmationRemoveGridRowTitle=Confirm remove
 confirmationRemoveGridRowMessage=Are you sure you want to remove this row?
 confirmationRemoveGridRowConfirm=Yes
 confirmationRemoveGridRowCancel=No
+confirmationVaccinationUpdate = Do you want to update the vaccination status?
```

Comment:
This is way too few information for the user. Which vaccination status? Update to what? This needs to be specified so that they can make an informed decision.